docs/Reference/ID-Prefix-Reservation.md

```diff
@@ -0,0 +1,156 @@
+---
+# required metadata
+
+title: ID Prefix Reservation Reference | Microsoft Docs
+author: dajaco
+ms.author: dajaco
+manager: unnir
+ms.date: 10/9/2017
+ms.topic: reference
+ms.prod: nuget
+#ms.service:
+ms.technology: null
+#ms.assetid: 
+
+# optional metadata
+
+description: Package ID Prefix Reservation feature description and author guide.
+keywords: NuGet package ID, prefix, reservation
+#ROBOTS:
+#audience:
+#ms.devlang:
+ms.reviewer:
+- anangaur
+- karann
+- unnir
+#ms.suite:
+#ms.tgt_pltfrm:
+#ms.custom:
+
+---
+
+# Package ID prefix reservation
+[NuGet.org](https://www.nuget.org/) and Visual Studio 2017 version 15.4 or later now show a visual indicator for packages that are submitted by owners with a reserved package ID prefix, as long as the package matches the reserved ID prefix naming pattern. The below reference explains what the ID prefix reservation entails, and how an owner can apply for an ID prefix.
```

Comment:
Docs shouldn't have a point in time references such as "now"

Reply:
Updated